CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -15,29 +15,25 @@ public class HarvestGateway : IHarvestDeveloperRetriever, ITimeSheetRetriever
         private readonly string _token;
         private readonly string _accountId;
         private readonly string _userAgent;
-        private readonly List<string> _listOfDeveloperRoles = new List<string>
-        {
-            "Software Engineer",
-            "Senior Software Engineer",
-            "Senior Engineer",
-            "Lead Engineer",
-            "Delivery Manager",
-            "SRE",
-            "Consultant",
-            "Delivery Principal"
-        };
-
+        private readonly string[] _listOfDeveloperRoles;
+       
         private static string ToHarvestApiString(DateTimeOffset date)
         {
             return date.ToString("yyyy-MM-dd");
         }
         
-        public HarvestGateway(string address, string token, string accountId, string userAgent)
+        private string[] CreateRoleArray(string roles)
```

Comment:
This can be static

Reply:
```suggestion
        private static string[] CreateRoleArray(string roles)
```

Reply:
@C-gyorfi you mean like the above?

Reply:
```suggestion
        private static string[] CreateRoleArray(string roles)
```

Reply:
@dushan-madetech Oh yes

Reply:
Resolved here https://github.com/chubberlisk/crypto_tech/pull/31/commits/b30487aaf172ffa8ec6ae6b08fc649711f62e2f7.